saleor/graphql/order/tests/mutations/test_draft_order_bulk_delete.py

```diff
@@ -1,8 +1,8 @@
 import graphene
 
-from ....order import OrderStatus
-from ....order import models as order_models
-from ...tests.utils import get_graphql_content
+from saleor.graphql.tests.utils import get_graphql_content
+from saleor.order import OrderStatus
+from saleor.order import models as order_models
```

Comment:
We use relative import in other test files. This takes place in several other files.
IMO we should use the same import type in all tests.